Samples/README.md

```diff
@@ -1,13 +1,136 @@
-   Replace the placeholder "<TOKEN_END_POINT_URL>" with the  bearer token endpoint which gives the bearerToken, implemented at your backend. 
\ No newline at end of file
+# skyflow-ios sdk sample templates
+Use this folder to test the functionalities of iOS-SDK just by adding `VAULT-ID` `VAULT-URL` and `SERVICE-ACCOUNT` details at the required place.
+
+### Prerequisites
+- iOS 13.0.0 and above
+- cocoapods
+- Xcode
+- [Node.js](https://nodejs.org/en/) version 10 or above
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 6.x.x
+
+- [express.js](http://expressjs.com/en/starter/hello-world.html)
+## Configure
+- Before you can run the sample app, create a vault.
+- `TOKEN_END_POINT_URL` for generating bearer token.
+
+
+### Create the vault
+1. In a browser, navigate to Skyflow Studio and log in.
+2. Create a vault by clicking **Create Vault** > **Start With a Template** > **PIIData**.
+3. Once the vault is created, click the gear icon and select **Edit Vault** Details.
+4. Note your Vault URL and Vault ID values, then click Cancel. You'll need these later.
+
+### Create a service account
+1. In the side navigation click, **IAM** > **Service Accounts** > **New Service Account**.
+2. For Name, enter **Test-Js-Sdk-Sample**. For Roles, choose the required roles for specific action.
+3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure, as you'll need it in the next steps.
+
+### Create TOKEN_END_POINT_URL
+- Create a new directory named `bearer-token-generator`.
+
+        mkdir bearer-token-generator
+- Navigate to `bearer-token-generator` directory.
+
+        cd bearer-token-generator
+- Initialize npm
+
+        npm init
+- Install `skyflow-node`
+
+        npm i skyflow-node
+- Create `index.js` file
+- Open `index.js` file
+- populate `index.js` file with below code snippet
+```javascript
+const express = require('express')
+const app = express()
+var cors = require('cors')
+const port = 3000
+const {
+    generateBearerToken,
+    isExpired
+} = require('skyflow-node');
+
+app.use(cors())
+
+let filepath = 'cred.json';
+let bearerToken = "";
+
+function getSkyflowBearerToken() {
+    return new Promise(async (resolve, reject) => {
+        try {
+            if (!isExpired(bearerToken)) resolve(bearerToken)
+            else {
+                let response = await generateBearerToken(filepath);
+                bearerToken = response.accessToken;
+                resolve(bearerToken);
+            }
+        } catch (e) {
+            reject(e);
+        }
+    });
+}
+
+app.get('/', async (req, res) => {
+  let bearerToken = await getSkyflowBearerToken();
+  res.json({"accessToken" : bearerToken});
+})
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`)
+})
+
+```
+- Start the server
+
+        node index.js
+    server will start at `localhost:3000`
```

Comment:
Delete line

Reply:
the empty line, here is for making `node index.js` as copyable code